conan/api/subapi/config.py

```diff
@@ -94,119 +97,42 @@ def install_pkg(self, ref, lockfile=None, force=False, remotes=None, profile=Non
-
-    def show(self, pattern):
-        return self.global_conf.show(pattern)
+        """ get the value of a global.conf item
+        """
+        return self._helpers.global_conf.get(name, default=default, check_type=check_type)
 
-    @property
-    def global_conf(self):
-        """ this is the new global.conf to replace the old conan.conf that contains
-        configuration defined with the new syntax as in profiles, this config will be composed
-        to the profile ones and passed to the conanfiles.conf, which can be passed to collaborators
+    def show(self, pattern) -> dict:
+        """ get the values of global.conf for those configurations that matches the pattern
         """
-        # Lazy loading
-        if self._new_config is None:
-            self._new_config = ConfDefinition()
-            self._populate_global_conf()
-        return self._new_config
-
-    def _populate_global_conf(self):
-        cache_folder = self.conan_api.cache_folder
-        new_config = self.load_config(cache_folder)
-        self._new_config.update_conf_definition(new_config)
-        if self._cli_core_confs is not None:
-            self._new_config.update_conf_definition(self._cli_core_confs)
+        return self._helpers.global_conf.show(pattern)
 
     @staticmethod
-    def load_config(home_folder):
-        # Do not document yet, keep it private
-        home_paths = HomePaths(home_folder)
-        global_conf_path = home_paths.global_conf_path
-        new_config = ConfDefinition()
-        if os.path.exists(global_conf_path):
-            text = load(global_conf_path)
-            distro = None
-            if platform.system() in ["Linux", "FreeBSD"]:
-                import distro
-            template = Environment(loader=FileSystemLoader(home_folder)).from_string(text)
-            home_folder = home_folder.replace("\\", "/")
-            content = template.render({"platform": platform, "os": os, "distro": distro,
-                                       "conan_version": conan_version,
-                                       "conan_home_folder": home_folder,
-                                       "detect_api": detect_api})
-            new_config.loads(content)
-        else:  # creation of a blank global.conf file for user convenience
-            default_global_conf = textwrap.dedent("""\
-                # Core configuration (type 'conan config list' to list possible values)
-                # e.g, for CI systems, to raise if user input would block
-                # core:non_interactive = True
-                # some tools.xxx config also possible, though generally better in profiles
-                # tools.android:ndk_path = my/path/to/android/ndk
-                """)
-            save(global_conf_path, default_global_conf)
-        return new_config
-
-    @property
-    def builtin_confs(self):
+    def conf_list():
+        """ list all the available built-in configurations
+        """
         return BUILT_IN_CONFS
```

Comment:
```suggestion
        return BUILT_IN_CONFS.copy()
```

So that user's can't modify this from under us and break things